__tests__/app.test.js

```diff
@@ -98,3 +98,163 @@ describe("GET /api/articles", () => {
+    const newComment = {
+      authorrr: "butter_bridge",
+      body: "master blaster",
+    };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid column value");
+      });
+  });
+  test("Responds with the correct error code when username does not exist", () => {
+    const newComment = {
+      author: "butterrrr_bridge",
+      body: "master blaster",
+    };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid key value insert");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("update article with object inc_votes: newVote and returns updated article", () => {
+    const patchVote = {
+      inc_votes: 5,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(patchVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: 105,
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("update article with object inc_votes: newVote, returns updated article with negative number value", () => {
+    const patchVote = {
+      inc_votes: -150,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(patchVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: -50,
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("Responds with the correct error code when username does not exist", () => {
+    const patchVote = {
+      inc_votessss: 5,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(patchVote)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid column value");
+      });
+  });
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("responds with status 204 and no content", () => {
+    return request(app)
+    .delete("/api/comments/3")
+    .expect(204)
+  })
+  test("responds with status 404 when the comment to delete doesn't exist", () => {
+    return request(app)
+    .delete("/api/comments/39999")
+    .expect(404)
+    .then(({body}) => {
+      console.log(body)
+      expect(body.msg).toBe("Comment not found")
+    })
+  })
+})
+
+// 404 in the right place but thing doesnt exist - could exist but doesnt
```

Comment:
Lets remove comments and console.logs before committing to keep our code tidy on the main branch 👍